src/main/java/underdogs/devbie/notice/domain/Notice.java

```diff
@@ -20,6 +22,7 @@
 import underdogs.devbie.exception.CreateFailException;
 
 @Entity
+@Table(indexes = @Index(name = "i_notice", columnList = "jobPosition, noticeType"))
```

Comment:
```suggestion
@Table(indexes = @Index(name = "i_notice", columnList = "noticeType, jobPosition"))
```
공고타입에 경우 기본필터옵션이라서 이쪽이 더 효율적일 것 같습니다.

Reply:
그렇군요!
카디널리티만 생각해서 순서를 배치했는데, 찾아보니 첫번째 인덱스 조건이 null이면 아예 인덱스를 타지 않는다고 하는군요.
👏